src/dataframes.jl

```diff
@@ -8,7 +8,8 @@ end
 # if it's an array of symbols, set the labels and return a Vector{Any} of columns
 function handle_dfs(df::AbstractDataFrame, d::KW, letter, syms::AbstractArray{Symbol})
     get!(d, :label, reshape(syms, 1, length(syms)))
-    Any[collect(df[s]) for s in syms]
+    ret = Any[collect(df[s]) for s in syms]
+    hcat(ret...)
```

Comment:
Yes, Plots, handles a Vector{Any} well, but that's the issue: this code `Any[collect(df[s]) for s in syms]` creates a Vector in julia 0.4 but a 1-row Matrix in julia 0.5, which is what causes the error. Replacing hcat with `vec` works.